src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -796,26 +800,60 @@ public function single_row( $item ) {
 			if ( $screen->in_admin( 'network' ) ) {
 				if ( $is_active ) {
 					if ( current_user_can( 'manage_network_plugins' ) ) {
-						$actions['deactivate'] = sprintf(
-							'<a href="%s" id="deactivate-%s" aria-label="%s">%s</a>',
-							wp_nonce_url( 'plugins.php?action=deactivate&amp;plugin=' . urlencode( $plugin_file ) . '&amp;plugin_status=' . $context . '&amp;paged=' . $page . '&amp;s=' . $s, 'deactivate-plugin_' . $plugin_file ),
-							esc_attr( $plugin_id_attr ),
-							/* translators: %s: Plugin name. */
-							esc_attr( sprintf( _x( 'Network Deactivate %s', 'plugin' ), $plugin_data['Name'] ) ),
-							__( 'Network Deactivate' )
-						);
+						if ( $has_active_dependents ) {
+							$dependent_names = WP_Plugin_Dependencies::get_dependent_names( $plugin_file );
+
+							$screen_reader_text = sprintf(
+								__( 'This plugin cannot be deactivated as %s depends on it.' ),
```

Comment:
Should this be "You cannot deactivate..." ?

Some other messages in the admin reads:

"You cannot delete a plugin while it is active on the main site"
"You cannot install because <a href="%1$s">WordPress %2$s</a> requires MySQL version %3$s or higher. You are running version %4$s"
"You must provide an email address"
"You can update from WordPress 6.3.3 to ..."

Reply:
Yep, these can be tweaked/improved before strings freeze, all the way thorough beta :)

Reply:
Updated the text as suggested. Thanks.